firebase/test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Disabled all usages of `any` inside the `test/` directory exclusively, since it's a common sense that  we'll have a lot of mocks and any usages in the tests. Let me know what do you think of this decision.